config/schema.error.yaml

```diff
@@ -337,6 +337,12 @@
     suggestion: 'Install Elasticsearch service'
     stage: deploy
     type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_WRONG_SEARCH_ENGINE_CONFIGURED:
```

Comment:
name of error sounds strange `DEPLOY_WRONG_SEARCH_ENGINE_CONFIGURED`
maybe just `DEPLOY_WRONG_SEARCH_ENGINE` ?

Reply:
Yes, DEPLOY_WRONG_SEARCH_ENGINE sounds better but we already have such error

Reply:
See my comment https://github.com/magento/ece-tools/pull/782/files#r489704435